roles/netbootxyz/templates/menu/talos.ipxe.j2

```diff
@@ -67,10 +68,16 @@ item {{ item.key }} ${space} {{ item.name }}
 choose --default ${talos_platform} talos_platform
 goto talos
 
+# https://www.talos.dev/latest/reference/kernel/
 :talos_boot
 isset ${talos_base_url} || set talos_base_url ${talos_mirror}/latest/download
 isset ${talos_config_url} && set talos_config talos.config=${talos_config_url} ||
-set boot_params page_poison=1 printk.devkmsg=on slab_nomerge slub_debug=P pti=on talos.platform=${talos_platform} ${talos_config} {{ kernel_params }}
+{# Edit releases.talos.custom_kernel_params in main.yml for custom kernel params #}
+{% if releases.talos.custom_kernel_params is defined and releases.talos.custom_kernel_params %}
+set boot_params {{ releases.talos.kernel_params }}
```

Comment:
Ah shoot, typo

Should be `set boot_params {{ releases.talos.custom_kernel_params }}`.